tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1678,6 +1687,42 @@ public void TestDescription()
                 Assert.Equal((float)0, column[2]);
                 Assert.Equal((float)4, column[3]);
             }
+
+            // Explicitly check the dateTimes column
+            DataFrameColumn dateTimeColumn = description.Columns[description.Columns.Count - 1];
+            Assert.Equal(dateTimeColumn.Name, dateTimes.Name);
+            Assert.Equal(4, dateTimeColumn.Length);
+            Assert.Equal((float)10, dateTimeColumn[0]);
+            Assert.Null(dateTimeColumn[1]);
```

Comment:
Why aren't we displaying `Max` and `Min` for DateTime?

Reply:
Because I don't have a great answer to "does `Min`/`Max` make sense for `DateTime`"? Couple reasons:
1. I know we can convert each `DateTime` value to `ticks` and find the `Min`/`Max` that way, but is that the generally accepted method everywhere? I figured users can run a LINQ query if they really want to find the max/min or write their own for loop. 
2. `Max` and `Min` are part of `DataFrameColumn.Computations`. There's other API here such as `Sum`, `Product` etc. I didn't think they made sense for `DateTime` values.

We can revisit this as part of #2784 . For now, this let's users call `Description` without throwing an exception so they can see other useful information about the `DataFrame`.